exec/java-exec/src/main/java/org/apache/drill/exec/work/foreman/QueryManager.java

```diff
@@ -1,4 +1,4 @@
-/**
+/*
```

Comment:
As far as I  understood, `final RunQuery runQuery` now is not used in `QueryManager`, it can be removed from class.